content/en/v3/admin/guides/multi-cluster/multi-cluster-example.md

```diff
@@ -0,0 +1,260 @@
+network-connection-check      kuberhealthy                  OK   
+```
+### Import Remote Prod Repo
+```bash
+# Make sure you're in the dev cluster
+kubectl config set-context $DEV_CONTEXT
+jx ns jx
+jx project import --url https://github.com/jx3rocks/jx3-gke-gsm.prd.git 
+```
+```
+we are now going to create a Pull Request on the development cluster git repository to setup CI/CD via GitOps
+
+created file /tmp/jx-git-643546451/.jx/gitops/source-config.yaml
+Created Pull Request: https://github.com/jx3rocks/jx3-gke-gsm.dev/pull/1
+
+we now need to wait for the Pull Request to merge so that CI/CD can be setup via GitOps
+
+Waiting up to 20m0s for the pull request https://github.com/jx3rocks/jx3-gke-gsm.dev/pull/1 to merge with poll period 20s....
+Pull Request https://github.com/jx3rocks/jx3-gke-gsm.dev/pull/1 was merged at sha 92e226e6e52cc96c1a84ba8dc455900d41cfe09d after waiting 2m1.698196955s
+
+waiting up to 20m0s for a trigger to be added to the lighthouse configuration in ConfigMap config in namespace jx for repository: jx3rocks/jx3-gke-gsm-prd
+you can watch the boot job to update the configuration via: jx admin log
+for more information on how this works see: https://jenkins-x.io/docs/v3/about/how-it-works/#importing--creating-quickstarts
+
+
+WARNING: It looks like the boot job failed to setup this project.
+You can view the log via: jx admin log
+error: failed to wait for repository to be setup in lighthouse: failed to find trigger in the lighthouse configuration in ConfigMap config in namespace jx for repository: jx3rocks/jx3-gke-gsm-prd within 20m0s
+error: failed to wait for the pipeline to be setup jx3rocks/jx3-gke-gsm-prd: failed to run 'jx pipeline wait --owner jx3rocks --repo jx3-gke-gsm-prd' command in directory '', output: ''
+```
+### Environments
+```bash
+# Development
+kubectl config set-context $DEV_CONTEXT
+kubectl get env
+NAME              NAMESPACE            KIND          PROMOTION   ORDER   GIT URL                                           GIT BRANCH
+dev               jx                   Development   Never               https://github.com/jx3rocks/jx3-gke-gsm.dev.git   master
+jx3-gke-gsm-prd   jx-jx3-gke-gsm-prd   Permanent     Auto        500     https://github.com/jx3rocks/jx3-gke-gsm.prd.git   master
+staging           jx-staging           Permanent     Auto        100     
+
+# Production
+kubectl config set-context $PROD_CONTEXT
+kubectl get env
+NAME      NAMESPACE    KIND          PROMOTION   ORDER   GIT URL                                           GIT BRANCH
+dev       jx           Development   Never               https://github.com/jx3rocks/jx3-gke-gsm.prd.git   master
+staging   jx-staging   Permanent     Auto        100           
+```
+### Set Dev Container Registry to Public
+In order to deploy applications to the remote prod environment it is required that the dev container registry is publically available. The image below highlights how you can make the change using the Google console.
```

Comment:
It's not mandatory. AFAIK the best way to do it is to add the dev registry secret to the prod cluster.

@jstrachan WDYT?

Reply:
That does sound like a better solution. Now to figure out the syntax to do it.